src/machines/modelingMachine.ts

```diff
@@ -1493,6 +1450,63 @@ export const modelingMachine = setup({
         return {} as SetSelections
       }
     ),
+    extrudeAstMod: fromPromise<
+      unknown,
```

Comment:
How come `unknown`? :o

Reply:
I think it's because I wanted this to be `void` but since I was misusing `Error` instead of `Promise.reject` this type was not what I expected it to be. Fixing your other feedback let me make this type `void` like I wanted it to be!